components/PoisDetails/Carousel.vue

```diff
@@ -63,4 +63,10 @@ export default defineNuxtComponent({
   border: 2px solid #fff;
   border-radius: 100%;
 }
+
+:deep(picture img) {
+  aspect-ratio: 16/9;
+  object-fit: cover;
+  width: 100%;
```

Comment:
I think only `width` have effect.  `aspect-ratio: 16/9;`. And `object-fit: cover;` it is already inherited from `.tw-object-cover`.

I am not sure if it is an other issue or not. But the download image size if not the right.

Reply:
After checking everything is needed !
- `aspect-ratio` constraint the frame width/height
- `width` tells the frame to take the full widht of parent column.
- `object-fit` tell how the image should cover the frame